packages/core/src/filters/Filter.ts

```diff
@@ -188,10 +189,31 @@ import type { FilterSystem } from './FilterSystem';
 export class Filter extends Shader
 {
     /**
-     * Default filter resolution for any filter.
-     * @static
+     * Override whether or not the resolution of the filter is automatically adjusted to match the resolution
+     * of the renderer.
+     * Setting this to false can allow you to get crisper filter at lower render resolutions.
+     * @example
+     * // renderer has a resolution of 1
+     * const app = new Application();
+     *
+     * Filter.defaultResolution = 2;
+     * Filter.defaultAutoResolution = false;
+     * // filter has a resolution of 2
+     * const filter = new GlowFilter();
+     */
+    public static defaultAutoResolution = true;
```

Comment:
The behavior here of `true` would be a breaking change since now the Filter, by default, with inherit from Renderer. This breaks from the current behavior of defaulting to 1. My suggestion is we set this to `false` and change it to `true` in our next major release.

Reply:
The other context about this is: in general, when faced with a design choice of performance vs being visually-rich, Pixi general prefers performance and opt-in to visually-richness (e.g., resolution, antialias, multisample). Filters can be expensive, especially at full Renderer resolution. That was one of the intentional decisions for defaulting to `1`. Unlike Text, filters "break" batching of Sprites (i.e., rendering with a slower, de-optimized way). For these reasons, I also think `false` is better out-of-the-box performance in many cases.